usagov/usagov-2021
# Usagov 1318 ceo result edit button behavior

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1318

## Description

Change the behavior of the "Edit my address" button in this page [https://www.usa.gov/elected-officials-results,](https://www.usa.gov/elected-officials-results) so it pre-populate with the address that was entered.
Person can still see what they entered, and see what needs to be edited. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Go to this page http://localhost/elected-officials-results and edit the content so it looks like this:
<img width="582" alt="image" src="https://github.com/usagov/usagov-2021/assets/39020809/f3271ca6-09a1-4087-b86e-4217bc119b88">

2. Go to http://localhost/elected-officials and check that all the fields are empty.
3. Enter any address you want and submit the form.
4. In the results page, click the "Edit my address" button.
5. Once you're in the http://localhost/elected-officials page, check that the URL looks similar to this: http://localhost/elected-officials?input-street=47+W+13th+St&input-city=New+York&input-state=NY&input-zip=10011 and that the form is pre-propulated
<img width="657" alt="image" src="https://github.com/usagov/usagov-2021/assets/39020809/98a36113-c8ba-4079-a1c8-ff1cd774faa2">


### Requires New Config

### Requires New Content

### Validation Steps
## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
